known_bugs.strings

```diff
@@ -898,7 +898,10 @@ ZNK4Item10field_typeEv
 ZN15Event_scheduler5startEPi                                                            ## https://bugs.launchpad.net/percona-server/+bug/1587195
 ZN7keyring10Vault_curl8read_keyERKNS_9Vault_keyEPNSt7__cxx1112basic_stringIcSt11char_traitsIcENS_16Secure_allocatorIcEEEE  ## https://jira.percona.com/browse/PS-264
 Z24plugin_foreach_with_maskP3THDPPFcS0_PP13st_plugin_intPvEijS4_                                                           ## https://jira.percona.com/browse/PS-264
-
+Z21copy_system_variablesP3THDb						        	## https://jira.percona.com/browse/PS-3817
+#ZN7myrocks10ha_rocksdb18get_auto_incrementEyyyPyS1_			       ## Fixed	## https://jira.percona.com/browse/MYR-235
+#ZN7myrocks10ha_rocksdb31update_auto_incr_val_from_fieldEv                     ## Fixed	## https://jira.percona.com/browse/MYR-235
```

Comment:
* Only use spaces in the file (not sure if these are spaces or tabs)
* There seem to be extra spaces or tabs on this line